src/SourceBuild/tarball/content/README.md

```diff
@@ -0,0 +1,152 @@
+
+- [Limitations](#limitations)
+- [Goals](#goals)
+- [Building the VMR](#building-the-vmr)
+- [List of components](#list-of-components)
+- [Filing Issues](#filing-issues)
+- [Useful Links](#useful-links)
+- [.NET Foundation](#net-foundation)
+- [License](#license)
+
+This repository is an **Experimental Virtual Monolithic Repository** that hosts all the code to help you build the entire .NET product end-to-end from source (often referred to as source-build).
+
+What this means:
+- **Monolithic** - a join of multiple individual repositories that make up the whole product, such as [dotnet/runtime](https://github.com/dotnet/runtime) or [dotnet/sdk](https://github.com/dotnet/sdk).
+- **Virtual** - not a replacement for the individual repositories that the product consists of but rather a mirror where sources of the individual repositories are synchronized into.
+- **Experimental** - not to be depended on as we reserve the right to delete the current instance and create a new, different one in its stead.
+
+In the VMR, you can find:
+- [source files of each product repository](#list-of-components) which are mirrored inside of their respective directories under [`src/`](src/),
+- tooling that enables [building the whole .NET product from source](https://github.com/dotnet/source-build) on Linux platforms,
+- small customizations, in the form of patches, applied on top of the original code to make the build possible,
+- *[in future]* E2E tests for the whole .NET product.
+
+More in-depth documentation about the VMR can be found in the [here (VMR Design And Operation)](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#layout).  
+See also the [dotnet/source-build](https://github.com/dotnet/source-build) repo for more information about the source-build itself.
+
+## Limitations
+**This is a work-in-progress and an experiment.**
+There are considerable limitations to what is possible at the moment. For an extensive list of current limitations, please see [Temporary Mechanics](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#temporary-mechanics).
+
+### Supported platforms / builds
+
+.NET Source-Build is supported on the oldest available .NET SDK feature update, and on Linux only.
+For example, if both .NET 6.0.1XX and 6.0.2XX feature updates are available from [dotnet.microsoft.com](https://dotnet.microsoft.com/en-us/download/dotnet/6.0), Source-Build will only support 6.0.1XX.
+For the latest information about Source-Build support for new .NET versions, please check our [GitHub Discussions page](https://github.com/dotnet/source-build/discussions) for announcements.
+
+It is expected that other platforms will be supported in the future.
+
+### Code flow
+For the time being, the source code only flows one way - from the individual repos into the VMR.
+Changes done to the VMR are not automatically mirrored back. More details on this process:
```

Comment:
I feel like we should provide clarity around the timeline of the code flow.  e.g. .NET 8.0 one way, .NET 9.0+ two way.  The linked documentation does not call this out either.